taskcluster/ci/browsertime/kind.yml

```diff
@@ -0,0 +1,229 @@
+---
+loader: taskgraph.loader.transform:loader
+transforms:
+    - fenix_taskgraph.transforms.browsertime:transforms
+    - fenix_taskgraph.transforms.notify:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - signing
+    - toolchain
+
+only-for-build-types:
+    - performance-test
+
+only-for-abis:
+    - armeabi-v7a
+    - arm64-v8a
+
+job-defaults:
+    dependencies:
+        geckoview-nightly: geckoview-nightly
+    notify:
+        by-level:
+            '3':
+                email:
+                    content: This calls for an action of the Performance team. Use the link to view it on Treeherder.
+                    link:
+                        text: Treeherder Job
+                        href: 'https://treeherder.mozilla.org/#/jobs?repo={product_name}&revision={head_rev}&searchStr={task_name}'
+                    on-reasons: [failed]
+                    subject: '[{product_name}] Raptor-Browsertime job "{task_name}" failed'
+                    to-addresses: [perftest-alerts@mozilla.com]
+            default: {}
+    run-on-tasks-for: []
+    treeherder:
+        kind: test
+        tier: 2
+        platform:
+            by-abi:
+                arm64-v8a: android-hw-p2-8-0-android-aarch64/opt
+                armeabi-v7a: android-hw-g5-7-0-arm7-api-16/opt
+    worker-type:
+        by-abi:
+            armeabi-v7a: t-bitbar-gw-perf-g5
+            arm64-v8a: t-bitbar-gw-perf-p2
+    worker:
+        max-run-time: 3600
+        env:
```

Comment:
Question: I wonder if we should factorize this payload into its own transform. This way both `browsertime` and `raptor` can have their common configuration at a single spot.

https://github.com/mozilla-mobile/fenix/blob/b0e4453fb7ba558a6308158e192c5ebfa9dfafb4/taskcluster/ci/raptor/kind.yml

What do you think @tomprince?

Reply:
I'm fine with either solution, but note that we're going to disable Raptor completely soon after this change lands (or almost, we'll still have the youtube-playback test running through it for a little longer, it still needs to be implemented in Browsertime).

Reply:
Ah, in that case no need to factorize. Let's keep browsertime this way!